homeassistant/components/alarmdecoder/__init__.py

```diff
@@ -121,6 +121,7 @@ def setup(hass, config):
     device = conf.get(CONF_DEVICE)
     display = conf.get(CONF_PANEL_DISPLAY)
     zones = conf.get(CONF_ZONES)
+    auto_bypass = conf.get(CONF_AUTO_BYPASS)
```

Comment:
Please use `dict[key]` for required config keys and keys with default config schema values.

Reply:
Done